docs/prover/diagnosis/index.md

```diff
@@ -0,0 +1,21 @@
+Diagnostic Tools in the Certora Prover
+======================================
+
+
+(tac-reports)=
+# TAC Reports
```

Comment:
hm, I'm not sure ...
things like unsat core could also be called "diagnostics" ... just checked -- it lives under "checking", which is meant as "checking how good the specs are" 

so, the answer is: I can't think of any other obvious subsections here right now; I made this category because it didn't fit so well with anything else (and I don't only want to talk about timeout tac reports, but also the SAT and UNSAT ones there, so I can't just put it with the timeouts)

 --> maybe I just change the header from Diagnosis to TAC Reports, and if there's more diagnosis, we add the more general header back?